Packs/cisco-meraki/Integrations/CiscoMerakiv2/CiscoMerakiv2.yml

```diff
@@ -0,0 +1,2283 @@
+      type: Number
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusAccountingServers.caCertificate
+      description: Certificate used for authorization for the RADSEC Server.
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusAccountingServers.host
+      description: IP address (or FQDN) to which the APs will send RADIUS accounting messages.
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusServers.openRoamingCertificateId
+      description: The ID of the Openroaming Certificate attached to radius server.
+      type: Number
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusServers.port
+      description: UDP port the RADIUS server listens on for Access-requests.
+      type: Number
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusServers.caCertificate
+      description: Certificate used for authorization for the RADSEC Server.
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusServers.host
+      description: IP address (or FQDN) of your RADIUS server.
+      type: String
+  - name: meraki-network-l3firewall-rule-list
+    description: List the L3 firewall rules for an MX network.
+    arguments:
+    - name: network_id
+      description: ID of the network to retrieve from. Use `meraki-network-list` to fetch all network IDs. This overrides the network ID instance parameter.
+      default: true
+    outputs:
+    - contextPath: CiscoMeraki.L3FirewallRule.networkId
+      description: ID of the network.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.comment
+      description: Comment for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.policy
+      description: Policy for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.protocol
+      description: Protocol for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.destPort
+      description: Destination port for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.destCidr
+      description: Destination CIDR for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.srcPort
+      description: Source port for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.srcCidr
+      description: Source CIDR for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.syslogEnabled
+      description: Whether syslog is enabled.
+      type: Boolean
+  - name: meraki-network-l3firewall-rule-update
+    description: Update the L3 firewall rules of an MX network.
+    arguments:
+    - name: network_id
+      description: ID of the network to update from. Use `meraki-network-list` to fetch all network IDs. This overrides the network ID instance parameter.
+      default: true
+    - name: override
+      description: Whether to override the current rules or append to them.
```

Comment:
I think this should be: Whether to override the current rules. If false, will append to the current rules.

Reply:
changed